### PR TITLE
Update tutorial with adc module

### PR DESCRIPTION
**Pull request by mestcihazal**
took out process and remotes parts and added module

### CODE REVIEW

**Comment by mestcihazal** [comment]
the wiring diagram needs updating from 22 gpio 25 to GPIO 8 (SPI Chip Select 0), which i will update as soon as cristina sends me the new wiring diagram

File: docs/tutorials/projects/make-a-plant-watering-robot.md
Comment (mestcihazal) [inline docs/tutorials/projects/make-a-plant-watering-robot.md:369]:
```suggestion
This module allows you to get multiple readings at the same time from different channels of the ADC sensor. If you wire and configure another sensor, such as temperature sensor on channel 1, you can add the sensor to the channel_map and get a reading from it.
```

**Review by mestcihazal — COMMENTED** [review]
fixed vale issue with 'so'

File: docs/tutorials/projects/make-a-plant-watering-robot.md
Comment (sguequierre) [inline docs/tutorials/projects/make-a-plant-watering-robot.md:354]:
```suggestion
The Viam registry (/extend/modular-resources/) allows hardware and software engineers to collaborate on their robotics projects by writing and sharing custom modules with each other.
You can add a module from the Viam registry directly from your robot’s Configuration tab in the Viam app, using the + Create component button.
```

File: docs/tutorials/projects/make-a-plant-watering-robot.md
Comment (sguequierre) [inline docs/tutorials/projects/make-a-plant-watering-robot.md:360]:
```suggestion
3. Add the necessary attributes.
For example, if you have a moisture sensor on channel 0, and your sensor_pin is 8, your configuration should look like this:
```
Not a major change at all we just try to have every sentence put on a newline in docs

File: docs/tutorials/projects/make-a-plant-watering-robot.md
Comment (sguequierre) [inline docs/tutorials/projects/make-a-plant-watering-robot.md:369]:
```suggestion
This module allows you to get multiple readings at the same time from different channels of the ADC sensor.
If you wire and configure another sensor, such as a temperature sensor on channel 1, you can add the sensor to the `"channel_map"` and get a reading from it.
```

**Review by sguequierre — COMMENTED** [review]
Couple of spacing comments and clarity suggestions otherwise looks great, TYSM for this contribution Hazal!

File: docs/tutorials/projects/make-a-plant-watering-robot.md
Comment (npentrel) [inline docs/tutorials/projects/make-a-plant-watering-robot.md:13]: This is interesting and a good update - I need to make some changes so that shows up as an option to select on the tutorials page - just fyi

File: docs/tutorials/projects/make-a-plant-watering-robot.md
Comment (npentrel) [reply docs/tutorials/projects/make-a-plant-watering-robot.md:354]: Link currently not working!

```suggestion
The [Viam registry](/extend/modular-resources/) allows hardware and software engineers to collaborate on their robotics projects by writing and sharing custom modules with each other. You can add a module from the Viam registry directly from your robot’s Configuration tab in the Viam app, using the + Create component button.
```

File: docs/tutorials/projects/make-a-plant-watering-robot.md
Comment (npentrel) [reply docs/tutorials/projects/make-a-plant-watering-robot.md:360]: (and indented to match)
```suggestion
3. Add the necessary attributes. 
    For example, if you have a moisture sensor on channel 0, and your sensor_pin is 8, your configuration should look like this:
```

File: docs/tutorials/projects/make-a-plant-watering-robot.md
Comment (npentrel) [reply docs/tutorials/projects/make-a-plant-watering-robot.md:369]: ^ this but also could we add a link to the module code? Something like:

```
{{< alert title="Info" color="info" >}}
If you would like to see how the module works, you can find its code on [GitHub](link to module code).
{{< /alert >}}
```

**Review by npentrel — COMMENTED** [review]
LGTM once Sierra approves, just a few fly-by comments

And please make the PR title in sentence case - that becomes the commit message and we try to keep those consistent :). I'll change it here.

File: docs/tutorials/projects/make-a-plant-watering-robot.md
Comment (mcvella) [inline docs/tutorials/projects/make-a-plant-watering-robot.md:32]:
```suggestion
4. [Configure the ADC as a module from the registry](#configure-the-adc-as-a-module-from-the-registry) along with the capacitive moisture sensor.
```
As the ADC itself is not a sensor per-se

File: docs/tutorials/projects/make-a-plant-watering-robot.md
Comment (mcvella) [inline docs/tutorials/projects/make-a-plant-watering-robot.md:169]: This whole section should be gone, as the modular resource abstracts this away

File: docs/tutorials/projects/make-a-plant-watering-robot.md
Comment (mcvella) [inline docs/tutorials/projects/make-a-plant-watering-robot.md:350]:
```suggestion
However, there are many different types of sensors used for sensing different things across the [Internet of Things](https://medium.com/@siddharth.parakh/the-complete-list-of-types-of-sensors-used-in-iot-63b4003ab6b3). Although the capacitive soil moisture sensor is not currently one of Viam's built-in models, you can add an analog-to-digital-converter (ADC)  as a module and use it to get readings from the moisture sensor.
```

File: docs/tutorials/projects/make-a-plant-watering-robot.md
Comment (mcvella) [inline docs/tutorials/projects/make-a-plant-watering-robot.md:344]:
```suggestion
### Configure the ADC as a module from the registry
```

**Review by mcvella — CHANGES_REQUESTED** [review]

File: docs/tutorials/projects/make-a-plant-watering-robot.md
Comment (mestcihazal) [reply docs/tutorials/projects/make-a-plant-watering-robot.md:13]: thank you!

File: docs/tutorials/projects/make-a-plant-watering-robot.md
Comment (mestcihazal) [reply docs/tutorials/projects/make-a-plant-watering-robot.md:369]: will add!

File: docs/tutorials/projects/make-a-plant-watering-robot.md
Comment (mestcihazal) [inline docs/tutorials/projects/make-a-plant-watering-robot.md:32]:
```suggestion
4. [Configure the ADC as a module from the registry](#configure-the-adc-as-a-module-from-the-registry).
```

File: docs/tutorials/projects/make-a-plant-watering-robot.md
Comment (mestcihazal) [inline docs/tutorials/projects/make-a-plant-watering-robot.md:371]:
```suggestion
If you wire and configure another sensor, such as a temperature sensor on channel 1, you can add the sensor to the `"channel_map"` and get a reading from it.

{{< alert title="Info" color="info" >}} If you would like to see how the module works, you can find its code on [GitHub](https://github.com/viam-labs/mcp300x-adc-sensor). {{< /alert >}} 
```

Comment (mestcihazal) [inline docs/tutorials/projects/make-a-plant-watering-robot.md:371]:
```suggestion
If you wire and configure another sensor, such as a temperature sensor on channel 1, you can add the sensor to the `"channel_map"` and get a reading from it.
{{< alert title="Info" color="info" >}} If you would like to see how the module works, you can find its code on [GitHub](https://github.com/viam-labs/mcp300x-adc-sensor). {{< /alert >}} 
```

File: docs/tutorials/projects/make-a-plant-watering-robot.md
Comment (mestcihazal) [inline docs/tutorials/projects/make-a-plant-watering-robot.md:447]: ### 
```suggestion
To tinker this example code to work best for you, determine at what [analog value from the soil moisture readings](#test-your-soil-moisture-readings-on-your-pi) you want to water your plant, as your thirsty plant's average moisture reading might differ from our example value of `60000`.
```

File: docs/tutorials/projects/make-a-plant-watering-robot.md
Comment (mestcihazal) [inline docs/tutorials/projects/make-a-plant-watering-robot.md:169]:
```suggestion
```

Comment (mestcihazal) [inline docs/tutorials/projects/make-a-plant-watering-robot.md:169]:
```suggestion
```

File: docs/tutorials/projects/make-a-plant-watering-robot.md
Comment (mestcihazal) [inline docs/tutorials/projects/make-a-plant-watering-robot.md:171]:
```suggestion
```

File: docs/tutorials/projects/make-a-plant-watering-robot.md
Comment (mestcihazal) [inline docs/tutorials/projects/make-a-plant-watering-robot.md:225]:
```suggestion
```

File: docs/tutorials/projects/make-a-plant-watering-robot.md
Comment (mestcihazal) [reply docs/tutorials/projects/make-a-plant-watering-robot.md:169]: i will rework this section in a different pr, i think there are some sections that could stay, like update/upgrading the pi, creating a plant-watering directory for the code, etc.

File: docs/tutorials/projects/make-a-plant-watering-robot.md
Comment (mestcihazal) [inline docs/tutorials/projects/make-a-plant-watering-robot.md:354]:
```suggestion
The [Viam registry](/extend/modular-resources/) allows hardware and software engineers to collaborate on their robotics projects by writing and sharing custom modules with each other.
```

File: docs/tutorials/projects/make-a-plant-watering-robot.md
Comment (mestcihazal) [inline docs/tutorials/projects/make-a-plant-watering-robot.md:361]:
```suggestion
3. Add the necessary attributes.
```

File: docs/tutorials/projects/make-a-plant-watering-robot.md
Comment (mestcihazal) [inline docs/tutorials/projects/make-a-plant-watering-robot.md:373]:
```suggestion
{{< alert title="Info" color="info" >}} If you would like to see how the module works, you can find its code on [GitHub](https://github.com/viam-labs/mcp300x-adc-sensor). {{< /alert >}}
```

**Review by mcvella — APPROVED** [review]

**Pull request merged**
